recipe/meta.yaml

```diff
@@ -94,9 +95,12 @@ outputs:
         - {{ compiler('c') }}
       host:
         - {{ pin_subpackage('libcurl', exact=True) }}
+        # OpenSSL is needed here to prevent conda-build from getting confused while
+        # both OpenSSL 1.1.1 and 3.0.0 and being supported
```

Comment:
This comment should be removed; it is confusing since it's not associated with any of the listed dependencies.

Reply:
Removed